scrapy/extensions/feedexport.py

```diff
@@ -46,6 +46,38 @@ def build_storage(builder, uri, *args, feed_options=None, preargs=(), **kwargs):
     return builder(*preargs, uri, *args, **kwargs)
 
 
+class ItemFilter:
+    """
+    This will be used by FeedExporter to decide if an item should be allowed
+    to be exported to a particular feed.
+
+    :param feed_options: feed specific options passed from FeedExporter
+    :type feed_options: dict
+    """
+
+    def __init__(self, feed_options):
+        self.feed_options = feed_options
+        self.item_classes = set()
+
+        if 'item_classes' in self.feed_options:
+            for item_class in self.feed_options['item_classes']:
+                self.item_classes.add(load_object(item_class))
+
+    def accepts(self, item):
+        """
+        Return ``True`` if `item` should be exported or ``False`` otherwise.
+
+        :param item: scraped item which user wants to check if is acceptable
+        :type item: scrapy supported items (dictionaries, Item objects, dataclass objects, and attrs objects)
```

Comment:
Should we have this list there? It's going to be outdated soon.

Reply:
Good point, it would be better to say “Scrapy items” and link to the items documentation.

Reply:
I'll update it accordingly.